src/routes/Apps/App/LogDialogs.tsx

```diff
@@ -1036,124 +455,61 @@ class LogDialogs extends React.Component<Props, ComponentState> {
+                            onChange={this.onChangeSearchString}
+                            onSearch={this.onSearch}
                         />
-                    </>
-
-
-                    <ChatSessionModal
-                        app={this.props.app}
-                        editingPackageId={this.props.editingPackageId}
-                        open={this.state.isChatSessionWindowOpen}
-                        onClose={this.onCloseChatSessionWindow}
-                    />
-                {
-                    teachSession && teachSession.teach &&
-                    <TeachSessionModal
-                        isOpen={this.state.isTeachDialogModalOpen}
-                        app={this.props.app}
-                        teachSession={teachSession}
-                        editingPackageId={this.props.editingPackageId}
-                        originalTrainDialogId={null}
-                        onClose={this.onCloseTeachSession}
-                        onSetInitialEntities={null}
-                        onEditTeach={(activityIndex, editHandlerArgs, tags, description, editHandler) => this.onEditTeach(activityIndex, editHandlerArgs ? editHandlerArgs : undefined, tags, description, editHandler)}
-                        onInsertAction={(trainDialog, activity, editHandlerArgs) => this.onInsertAction(trainDialog, activity, editHandlerArgs.isLastActivity!)}
-                        onInsertInput={(trainDialog, activity, editHandlerArgs) => this.onInsertInput(trainDialog, activity, editHandlerArgs.userInput)}
-                        onDeleteTurn={(trainDialog, activity) => this.onDeleteTurn(trainDialog, activity)}
-                        onChangeExtraction={(trainDialog, activity, editHandlerArgs) => this.onChangeExtraction(trainDialog, activity, editHandlerArgs.extractResponse, editHandlerArgs.textVariations)}
-                        onChangeAction={(trainDialog, activity, editHandlerArgs) => this.onChangeAction(trainDialog, activity, editHandlerArgs.trainScorerStep)}
-                        onEndSessionActivity={this.onEndSessionActivity}
-                        onReplayDialog={(trainDialog) => this.onReplayTrainDialog(trainDialog)}
-                        editType={this.state.editType}
-                        initialHistory={this.state.activityHistory}
-                        lastAction={this.state.lastAction}
-                        sourceTrainDialog={this.state.currentTrainDialog}
-                        allUniqueTags={this.props.allUniqueTags}
+                    </div>
+                    <OF.DetailsList
+                        data-testid="logdialogs-details-list"
+                        key={this.state.detailListKey}
+                        className={`${OF.FontClassNames.mediumPlus} ${isPlaceholderVisible ? 'cl-hidden' : ''}`}
+                        items={computedLogDialogs}
+                        selection={this.selection}
+                        getKey={getDialogKey}
```

Comment:
Can't tell from all the moving but seems like you changed `dialogKey` to `detailsListKey` and likewise should change `getDialogKey` to `getDetailsListKey`

Reply:
As discussed they are unrelated items - component key and line item key.  I renamed one to avoid confusion